interface-spec.md

```diff
@@ -38,9 +43,7 @@ TODO: to what extent should we use typed signatures (`.equals(Node other)`) vers
 
 **Properties:**
 
-- `String .value` blank node name as a string (example: `_:blank3`)
-
-TODO: Does the value always start with an underscore?
+- `String .value` blank node name as a string without leading `_:` (example: `blank3`)
```

Comment:
Any proposals for the wording? Should we replace blank node name with blank node identifier?